packages/pn-pa-webapp/src/pages/components/NewNotification/Recipient.tsx

```diff
@@ -562,7 +562,7 @@ const Recipient = ({
                         />
                       )}
                     </Grid>
-                    <Grid container spacing={2}>
```

Comment:
This change breaks physical address spacing

Reply:
Restored physical address spacing